### ISSUE TITLE
flutter release process and the VERSION file

### ISSUE THREAD

**Issue by devoncarew** [flutter/flutter#9887]
From the flutter release process docs here: https://github.com/flutter/flutter/wiki/Release-process

it looks like `packages/flutter/pubspec.yaml` will be the system of record for the flutter framework version. IntelliJ had been using the `/VERSION` file, but that file hasn't really been maintained.

Should tooling look in `packages/flutter/pubspec.yaml` or in `/VERSION` as the system-of-record for the flutter sdk version? It seems a bit odd to use a pubspec file inside the repo as the main version when there's a file at the root, but we're happy to use either - it would just be good to have an indication about which to use.

(the next released version of the plugin will be able to handle a flutter sdk w/ and w/o a `VERSION` file at the root)

**Comment by abarth** [comment]
Those version are for two different things:

 1. `packages/flutter/pubspec.yaml` describes `package:flutter`, which is one piece of the overall SDK.  For example, if we renamed `Widget` to `XYZ`, we'd adjust that version to say that we made a big change to `package:flutter`.

 2. `VERSION` describes the version of the Flutter SDK itself.  For example, if we renamed the `packages` directory to `pkgabc`, we'd adjust that version to say we made a big change to the structure of the SDK, even if all the code in all the packages remained the same.

I suspect it is appropriate for the IntelliJ plugin to look at `VERSION` because most of what it cares about is the structure of the SDK itself rather than the API exposed by any particular package.
